### ISSUE TITLE
Make the path shorten

### ISSUE THREAD

**Issue by Cyrusky** [omar-dulaimi/prisma-joi-generator#3]
## Problem

The generated pathes are tooooooooo looooooong, so please shorten them.

## Suggested solution

Maybe we can use a single letter to be the folder name, then give the best practice document for the definition of the folder.. just a thouths.

**Comment by Cyrusky** [comment]
And combine several similar functions to one file. then we do not need to import different components in the head of the file that has loooooong name.

**Comment by omar-dulaimi** [comment]
Hey @Cyrusky 

Thanks for the great suggestion. After the implementation, here's how the imports will look:

```ts
import { PostCreateSchema, UserAggregateSchema } from '@schemas';
```

As you can see, all schemas will get imported using a single import statement, single line.

There will be a need for a change on your side too(I will document it in the README).

Basically, you need to add paths in your `tsconfig` file like this:

```json
{
  "compilerOptions": {
    "paths": {
      "@schemas": ["./src/generated/joi"]
     }
  }
}
```

Would this fix your issues?

Please let me know so I get start working on it.

**Comment by omar-dulaimi** [comment]
Fixed in `0.1.1`

**Comment by Cyrusky** [comment]
Thx. let me try.
